lib/utilities/get-config.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+let Yam = require('yam');
+let Project = require('../models/project');
+
+module.exports = function getConfig(configOverride) {
+  if (configOverride) {
+    return configOverride;
+  }
+  return new Yam('ember-cli', {
```

Comment:
should we load a new Yam on each invocation of getConfig, or memoize?

Reply:
@stefanpenner I'd initially tried have the `new Yam` line at the module level but had to refactor because tests which relied on mocking the config failed (pod tests I believe). I can't remember the specific reasoning but I'll have another go and see.